Vic2ToHoI4/Data_Files/Configurables/cultureGroupToGraphics.txt

```diff
@@ -0,0 +1,1513 @@
+# Copyright (c) 2019 The Paradox Game Converters Project
```

Comment:
This file is upload to a capital C Configurables folder instead of the existing lower case one. This has confused github and will cause more problems in the future. Please rename the folder.